mycroft/skills/wolfram_alpha/__init__.py

```diff
@@ -184,7 +186,15 @@ def handle_fallback(self, message):
 
             self.speak(response)
         else:
-            self.speak("Sorry, I don't understand your request.")
+            if len(others) == 1:
```

Comment:
Since `others` is an array, we probably should do a verification of > 0 instead of equals 1.
`if len(others) > 0:`